src/Umbraco.Web.UI.Client/src/packages/sysinfo/repository/sysinfo.repository.ts

```diff
@@ -78,11 +72,10 @@ export class UmbSysinfoRepository extends UmbRepositoryBase {
 	}
 
 	async #getVersionCheckPeriod(): Promise<number> {
-		if (!this.#serverConfiguration) {
-			this.#serverConfiguration = await this.requestServerConfiguration();
-		}
-
-		return this.#serverConfiguration?.versionCheckPeriod ?? 7;
+		const appContext = await this.getContext(UMB_APP_CONTEXT);
+		return this.observe(appContext.getServerConnection().versionCheckPeriod, (period) => {
```

Comment:
Could we make a `getVersionCheckPeriod()` method, so we can avoid observing here?

Reply:
Or is it because the value may be undefined at this stage? ... :-/

Reply:
Then if you remove the callback, then the observation-controller will be destroyed once the promise has resolved. then that is not running in the background.. :-)

Reply:
I also see you may have thought that the callback sets what the promise resolves to, but they are not connected... But I see that they could have been.. :-D